compiler/rustc_lint_defs/src/builtin.rs

```diff
@@ -2967,13 +2967,13 @@ declare_lint_pass! {
         MISSING_ABI,
         INVALID_DOC_ATTRIBUTES,
         SEMICOLON_IN_EXPRESSIONS_FROM_MACROS,
-        DISJOINT_CAPTURE_MIGRATION,
+        RUST_2021_INCOMPATIBLE_CLOSURE_CAPTURES,
         LEGACY_DERIVE_HELPERS,
         PROC_MACRO_BACK_COMPAT,
-        OR_PATTERNS_BACK_COMPAT,
+        RUST_2021_INCOMPATIBLE_OR_PATTERNS,
         LARGE_ASSIGNMENTS,
-        FUTURE_PRELUDE_COLLISION,
-        RESERVED_PREFIX,
+        FUTURE_PRELUDE_COLLISIONS,
```

Comment:
```suggestion
        RUST_2021_PRELUDE_COLLISIONS,
```

this seems like another case where we should stick to the `rust_2021` convention. Surely there will be more prelude changes in future editions.